src/ert/config/gen_kw_config.py

```diff
@@ -224,6 +224,32 @@ def _check_valid_triangular_parameters(prior: PriorDict) -> None:
                     ).set_context(self.name)
                 )
 
+        def _check_valid_derrf_parameters(prior: PriorDict) -> None:
```

Comment:
It might be outside the scope for this PR, but did you consider converting this from a function to a dataclass? Something like:

```python
@pydantic.dataclass
class DERRF:
    name: str
    nbins: PositiveInt
    min: float
    max: float
    skewness: float
    width: confloat(gt=0)

    @model_validator(mode="after")
    def validate_min_max(self) -> Self:
        if not self.max > self.min
            raise ValueError(f"Max ({self.max}) must be larger than max ({self.max})
        return self

Reply:
Yes, I guess it will, though it would be possible to work around it for now, and only migrate once we do something like this for all parameter types. If that is the direction you are heading with rewriting parameter configs?